llvm/lib/CodeGen/MachineScheduler.cpp

```diff
@@ -3503,6 +3511,9 @@ void GenericScheduler::initPolicy(MachineBasicBlock::iterator Begin,
     RegionPolicy.OnlyBottomUp = false;
     RegionPolicy.OnlyTopDown = false;
   }
+
+  this->BotIdx = NumRegionInstrs - 1;
```

Comment:
we don't usually add `this` unless there is a naming confusion. Is there any local variable with the same name?

Reply:
there is for `NumRegionInstrs` but not `BotIdx`, removed thanks 👍